css/main.css

```diff
@@ -272,6 +282,232 @@ nav {
   height: 850px;
 }
 
+/* Sponsors */
+.sponsors {
+  position: relative;
+  width: 100%;
+  margin: 0 auto;
+  color: #ffad0b;
+  font-family: "Avenir Light", sans-serif;
+}
+
+.sponsors .cropped {
+  position: relative;
+  width: 1440px;
```

Comment:
Can we set this width to 100% for now so it doesn't mess up testing on the full-site branch?